R/drive_delete.R

```diff
@@ -1,5 +1,6 @@
 #' Delete file from Google Drive.
 #'
+#' @description Caution: this will permanently delete your files.
```

Comment:
I think we should only use the `@description` tag when necessary, such as when we're trying to include text after some bullet points in the same section.